packages/tools/src/tools/annotation/AngleTool.ts

```diff
@@ -795,6 +796,61 @@ class AngleTool extends AnnotationTool {
         }
       );
 
+      const center = canvasCoordinates[1];
+
+      // Calculate radius as 1/3 of the minimum distance from center to lines
+      const radius =
```

Comment:
could you please some configurations? enabled/disable, and maybe the offset from the joint to draw angles

Reply:
Done